spec/helpers/degrees_helper_spec.rb

```diff
@@ -89,15 +89,11 @@
   end
 
   describe "#countries_options" do
-    before do
-      stub_const("Dttp::CodeSets::Countries::MAPPING", { "country" => { country_code: "C" } })
-    end
-
     it "iterates over array and prints out correct countries values" do
-      expect(countries_options.size).to be 2
+      expect(countries_options.size).to be 281
```

Comment:
this will probably turn out to be somewhat fragile, as it's dependent on the list of countries in the reference gem. i think this test would be better if we stubbed `DfE::ReferenceData::CountriesAndTerritories::COUNTRIES_AND_TERRITORIES`.